ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -47,6 +47,27 @@ tests:
   commands: make test-unit
   container:
     from: src
+- as: ci-e2e
+  steps:
+    cluster_profile: aws
+    env:
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: numaresources-operator
+      OO_PACKAGE: numaresources-operator
+      OO_TARGET_NAMESPACES: '!all'
```

Comment:
probably it is not correct, and we should not install pao under all namespace, but we should first fix it under the numaresource-operator

Reply:
changed to '!install'